routes/_components/LazyImage.html

```diff
@@ -1,24 +1,40 @@
-<div class="lazy-image" style={computedStyle} >
+<div class="lazy-image {fillFixSize ? 'fixed-size': ''}" style={computedStyle} >
   <img
-    class={className}
     aria-hidden={ariaHidden}
     {alt}
     {title}
     {width}
     {height}
     src={displaySrc}
+    style={focusStyle}
     ref:node
   />
 </div>
 <style>
   .lazy-image {
+    margin: 0;
+    padding: 0;
     overflow: hidden;
+    display: flex;
   }
+
+  .fixed-size img {
```

Comment:
I would also prefer to add another class here (e.g. `fixed-size-img`) since it's a more performant selector